### PR TITLE
Ndlc changes v3reset

### PR DESCRIPTION
**Pull request by emb03**
@z3cka here it is again on a new branch but doesn't seem to have all the files. Did what you said to do.

### CODE REVIEW

**Comment by emb03** [comment]
@z3cka This seems to be right. I have two commits above that contain the changes I want.

File: pages/02.ndlc-schedule/program.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
+
 ---
-title: 'Tentative  Schedule'
+title: 'Schedule'
 published: true
```
Comment (z3cka) [inline pages/02.ndlc-schedule/program.md:3]: is it "Schedule" or "Tentative Schedule"?

**Comment by emb03** [comment]
@z3cka According to the Word Document that Jade sent, it is Tentative Schedule.

File: pages/02.ndlc-schedule/program.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
+
```
Comment (z3cka) [inline pages/02.ndlc-schedule/program.md]: oh!!! this is probably causing the strange frontmatter rendering difference

**Pull request merged**